samples/pmic/npmx/charger_and_events/README.rst

```diff
@@ -0,0 +1,97 @@
+.. _npmx_charger_and_events_sample:
+
+npmx: CHARGER AND EVENTS
+########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The npmx CHARGER AND EVENTS sample demonstrates how to control PMIC CHARGER and receive events using npmx drivers.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf52840dk_nrf52840, nrf5340dk_nrf5340_cpuapp
+
+The sample also requires an nPM1300 EK.
+
+Overview
+********
+
+This sample uses CHARGER and EVENTS driver to do the following:
+
+  * Set CHARGER basic configuration.
+  * Register EVENTS from device.
+
+Wiring
+******
+
+#. Connect the TWI interface between the chosen DK and the nPM1300 EK as in the following table:
+
+.. list-table:: nPM1300 EK connections.
+   :widths: 25 25 25
+   :header-rows: 1
+
+   * - nPM1300 EK
+     - nRF5340 DK
+     - nRF52840 DK
+   * - GPIO0
+     - P1.10
+     - P1.10
+   * - SDA
+     - P1.02
+     - P0.26
+   * - SCL
+     - P1.03
+     - P0.27
+   * - VOUT2 & GND
+     - External supply (P21)
+     - External supply (P21)
+
+#. Make the following connections on the nPM1300 EK:
+
+   * Connect a USB power supply to the **J3** connector.
+   * Connect a suitable battery to the **J2** connector.
+   * On the **P18** pin header, connect **VOUT2** and **VDDIO** pins with a jumper.
+   * On the **P2** pin header, connect **VBAT** and **VBATIN** pins with a jumper.
+   * On the **P17** pin header, connect all LEDs with jumpers.
+   * On the **P13** pin header, connect **RSET1** and **GND** pins with a jumper.
+   * On the **P14** pin header, connect **RSET2** and **VSET2** pins with a jumper.
+
+#. Make the following connections on the DK:
+
+   * Switch nRF power source **SW9** to **VDD**.
+   * Switch VEXT -> nRF **SW10** to **ON**.
+
+Configuration
+*************
+
+|config|
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/pmic/npmx/charger_and_events`
+
+.. include:: /includes/build_and_run.txt
+
+Testing
+=======
+
+After programming the sample to your development kit, connect and disconnect battery and USB-C cable to see various event messages.
+Battery charging status will also be printed.
+
+#. |connect_kit|
+#. |connect_terminal|
+#. Use `Charging current` in menuconfig to change charging current.
+#. Use `Termination voltage` in menuconfig to change Vterm voltage.
```

Comment:
```suggestion
#. Use the `Charging current` option in menuconfig to change charging current.
#. Use the `Termination voltage` option in menuconfig to change Vterm voltage.
```

Just to confirm - are both steps obligatory or you can choose only one of them?

Reply:
The user can change both, one, or none of the options.